viewer-prototype/src/browser/trace-explorer/trace-explorer-commands.ts

```diff
@@ -0,0 +1,21 @@
+import { Command, MenuPath } from '@theia/core';
+
+export namespace TraceExplorerMenus {
+    export const PREFERENCE_EDITOR_CONTEXT_MENU: MenuPath = ['trace-explorer-opened-traces-context-menu'];
+}
+export namespace TraceExplorerCommands {
+    export const OPEN_TRACE: Command = {
+        id: 'trace-explorer:preferences.open-trace',
```

Comment:
Is 'preferences' appropriate in these id's?

Reply:
Updated. Thanks.